### DISCUSSION TITLE
Is there a way to reset my password

### DISCUSSION THREAD

**Discussion by brobzilla** [pockethost/pockethost#86] [Q&A] [answered]
I created an account with pockethost but I've replaced the password  with the admin pocketbase password in my password manager.  

Thanks
Rob

**Comment by benallfree** [comment]
Hey Rob are you looking to reset your pocketbase admin password, or your pockethost password?

Either way, you raise a great point. I'm not sure either one is available, but if you want to send a PR before I get to it you're definitely welcome to do so! The pocketbase admin pw reset is more challenging because SMTP needs to be working on the instnace you want to reset.

**Reply by brobzilla** [reply]
Hey sorry about the confusing question.  It's my pockethost password.   I believe pocketbase has a forgot password link.

**Reply by benallfree** [reply]
pocketbase does have a 'forgot password' link, but the instances do not have SMTP configured by default so it's unlikely you'd receive a pw reset email.

Since you're looking to reset your pockethsot password and we don't have a way to do that, I'm converting this to an issue. Feel welcome to send a PR if you get to it before I do :)

**Comment by benallfree** [comment, marked as answer]
Closed by #94
